src/FSharp.Data.Fred/JsonApi.fs

```diff
@@ -1,668 +1,356 @@
-        /// `DateTime(yyyy,MM,dd)` formatted `DateTime`, optional, default: today's date.
-        /// </param>
-        /// <returns>A collection of fields describing the series release.</returns>
-        member this.Release(id:string,?realtimeStart:DateTime,?realtimeEnd:DateTime) =
-            let realtimeStart =
-                let dt = defaultArg realtimeStart DateTime.Now
-                dt.ToString("yyyy-MM-dd")
-            let realtimeEnd = 
-                let dt = defaultArg realtimeEnd DateTime.Now
-                dt.ToString("yyyy-MM-dd")
-            let queryParameters = [
-                "series_id", id.ToUpper()
-                "realtime_start", realtimeStart
-                "realtime_end", realtimeEnd
-            ]
+        Helpers.request key Endpoint.SeriesCategories queryParameters
+        |> Json.SeriesCategoriesResponse.Parse
+    member this.Release(id:string,?realtimeStart:DateTime,?realtimeEnd:DateTime) =
+        let realtimeStart =
+            let dt = defaultArg realtimeStart DateTime.Now
+            dt.ToString("yyyy-MM-dd")
+        let realtimeEnd = 
+            let dt = defaultArg realtimeEnd DateTime.Now
+            dt.ToString("yyyy-MM-dd")
+        let queryParameters = [
+            "series_id", id.ToUpper()
+            "realtime_start", realtimeStart
+            "realtime_end", realtimeEnd
+        ]
             
-            Helpers.request key Endpoint.SeriesRelease queryParameters
-            |> SeriesReleaseResponse.Parse 
+        Helpers.request key Endpoint.SeriesRelease queryParameters
+        |> Json.SeriesReleaseResponse.Parse 
 
 
-        /// <summary>
-        /// Get the FRED tags for a series.
-        /// </summary>
-        /// <param name="id">The id for a series. String, required.</param>
-        /// <param name="realtimeStart">
-        /// The start of the real-time period. For more information, see [Real-Time Periods](https://fred.stlouisfed.org/docs/api/fred/realtime_period.html).
-        /// `DateTime(yyyy,MM,dd)` formatted `DateTime`, optional, default: today's date.
-        /// </param>
-        /// <param name="realtimeEnd">
-        /// The start of the real-time period. For more information, see [Real-Time Periods](https://fred.stlouisfed.org/docs/api/fred/realtime_period.html).
-        /// `DateTime(yyyy,MM,dd)` formatted `DateTime`, optional, default: today's date.
-        /// </param>
-        /// <param name="orderBy">
-        /// Order results by values of the specified attribute.
-        /// One of the following: 
-        /// `OrderByTags.SeriesCount`, `OrderByTags.Popularity`, `OrderByTags.Created`, `OrderByTags.Name`, `OrderByTags.GroupId`.
-        /// optional, default: `OrderByTags.SeriesCount`
-        /// </param>
-        /// <param name="sortOrder">
-        /// Sort results is ascending or descending order for attribute values specified by `orderBy`.  
```

Comment:
As long as the SortOrder DU is explained, we can probably leave out this XML comment, because the docs provide a link to DU where it will be explained. The same would apply for other obvious parameters.